tests/puppeteer/campaigns/data/FO/cart.js

```diff
@@ -1,8 +1,15 @@
 module.exports = {
 
   customCartData: {
-    name: 'T-shirt imprimé colibri',
-    quantity: '1',
-    price: '22,94 €',
+    firstProduct: {
+      name: 'T-shirt imprimé colibri',
```

Comment:
You should test in english, so Product name should be in english
```suggestion
      name: 'Hummingbird printed t-shirt',
```